pyglmnet/datasets.py

```diff
@@ -0,0 +1,35 @@
+"""
+A set of convenience functions to download datasets for illustrative examples
+"""
+import subprocess
+import pandas as pd
+
+
+def fetch_tikhonov_data(dpath='/tmp/glm-tools'):
+    cmd = 'rm -rf ' + dpath
+    subprocess.call(cmd, shell=True)
```

Comment:
if you do this, windows users won't be able to use your scripts. The clean solution is to use `urllib` ...